### PR TITLE
deleted coverage'

### PR DESCRIPTION
**Pull request by Yoyo2Code**

### CODE REVIEW

File: app/controllers/admin/properties_controller.rb [hunk 1]
```diff
@@ -1,4 +1,23 @@
...
+
+  def new
+    @property = current_user.business.properties.new
+    @locations = Location.all
+  end
+
+  def create
+    location = Location.find(params[:location])
+    property = current_user.business.properties.new(property_params)
+    property.location = location
```
Comment (dtinianow) [inline app/controllers/admin/properties_controller.rb]: This still smells funny

File: app/views/admin/properties/new.html.erb [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    <br>
+    <%= f.label :price_per_guest %><br>
+    <%= f.number_field :price_per_guest, placeholder: "Price Per Guest" %><br>
+    <br>
+    <%= f.label :image_path %><br>
+    <%= f.text_field :image_path, placeholder: "Image Url" %><br>
+    <br>
+    <%= f.label :location %><br>
+    <%= select_tag "location", options_from_collection_for_select(@locations, "id", "city") %><br>
+    <%= f.label :max_occupancy %><br>
```
Comment (dtinianow) [inline app/views/admin/properties/new.html.erb]: Add a <br> here

File: spec/models/cart_items_spec.rb [hunk 3]
```diff
@@ -54,4 +54,29 @@
...
+                starting_date: "08/15/2016",
+                end_date: "09/01/2016"
+             }
+    cart_item = CartItem.new( params[:property_id],
+                              params[:occupancy],
+                              params[:starting_date],
+                              params[:end_date])
+
+    assert_equal 17, cart_item.night_count
+  end
```
Comment (dtinianow) [inline spec/models/cart_items_spec.rb]: Nice work here

**Comment by dtinianow** [comment]
Nice job resolving all those conflicts

**Comment by dtinianow** [comment]
Also good fix for creating a night count

**Comment by Automatic365** [comment]
That is a great fix

**Pull request merged**